client/stylesheets/puzzle.scss

```diff
@@ -280,6 +280,16 @@ table.puzzle-list {
   }
 }
 
+.chat-input-row {
+  display: flex;
```

Comment:
I believe this needs flex:none; to render as intended in Safari.

Reply:
Fixed via https://github.com/deathandmayhem/jolly-roger/commit/f40316407639ef2b5f942ed9f98411ab229f92bf -- thanks for testing and coming up with the fix!